### ISSUE TITLE
Ability to save link in a custom field

### ISSUE THREAD

**Issue by openbayou** [kraftbj/press-this-extended#23]
It would be awesome if this plugin had the ability to save a link in a custom field.

**Comment by CKMacLeod** [comment]
Agreed (if I understand you correctly): The ability to save the source-link separately would be very useful, and is frequently a feature of other bookmarklets. It's "almost there" in Press This. If I understood PressThis itself just a little better,I think I'd have it already.

Easiest way to achieve it that I've found in the meantime is sub-optimal. Add a function to save_post or wp_insert_post hooks, narrowed to a category set aside for link-posts, so that, on save/publish/update, the link is added to Post Meta via a preg-match of the post content. Much better (since it wouldn't require using the post editor), and more elegant since you're preg-matching for a variable already found in Press This than added to the "Source" link, would be a function that hooked into or filtered the post data array directly, possibly via wp_insert_post, and automatically added the key-value pair to Post Meta. 

Also nice, and somewhat similar, but buried even more deeply into Press This or absent, would be ability to grab other source info where present in Open Graph or other meta tags (especially original author name), and also save it as Post Meta or pre-embed it the Post Title and/or Content. . 

I'd wanted to avoid having to duplicate and fork Press This. This (excellent) plug-in points to the possibility of instead extending it rather elegantly. I'm just not sure whether it's possible to get the full desirable functionality through extension rather than forking.
